### DISCUSSION TITLE
Build works on Android, but Crash on Real device.

### DISCUSSION THREAD

**Discussion by OscarLeif** [scottcgi/Mojoc#22] [General]
Hello I have this doubt I try to run the Demo SuperLitle red on Android device.
But the game Crash. The Only clue that Android Studio show is when it Try to read the file "Font/hp.png"
Not sure if the files are not loaded before.

**Comment by scottcgi** [comment]
There is too little information, I find it difficult to determine the problem. There is an APK file in the repository, which is a Debug version packaged with AS. You can try it out — https://github.com/scottcgi/Mojoc/tree/master/Samples/Apk. Perhaps you can use the real machine debugging function of AS to see where the problem lies.
